examples/nextjs-webhook-stripe/README.md

```diff
@@ -0,0 +1,37 @@
+# Upstash Workflow Nextjs Example
+
+This project has some routes showcasing how Upstash Workflow can be used in a nextjs project. You can learn more in [Workflow documentation for Nextjs](https://upstash.com/docs/qstash/workflow/quickstarts/vercel-nextjs).
+
+Under the app directory, you will find 7 folders, each corresponding to a workflow API except the `-call-qstash`. In each of these folders, you will find the `route.ts` file which defines the endpoint.
```

Comment:
Can we update the readme? It would be good to have information about the routes. And about how we can make it work in development